cibuildwheel/environment.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, name: str, value: str):
     def __repr__(self) -> str:
         return f"{self.name}: {self.value}"
 
-    def evaluated_value(self, **kwargs: Any) -> str:
+    def evaluated_value(self, **_: Any) -> str:
```

Comment:
`**_kwargs` is fine if you like that better, I think it was complaining about it being unused. `**kwargs` means you are using the extra arguments, usually - the convention is arguments that start with an underscore might be unused, but normal arguments always will be used. It could also be an ignore comment, but I think `_` or `_kwargs` conveys the "we won't use it" meaning better than a comment and an unused `kwargs`. But not strongly opinionated on it; it's so short it's hard to miss that it's unused.

Reply:
Pylance (which is running in vscode) likes `**_` much better than `**kwargs` or `**_kwargs`. The later seems buggy to me, since leading underscore for unused args is a common convention.

Reply:
Yeah, I think `_kwargs` seems okay as a compromise. `**_` looks like a magic incantation of punctuation, like Perl or something :P